packages/odyssey-react/src/components/Tag/Tag.theme.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Color
+  ItemBackgroundColor: theme.ColorPaletteNeutral100,
+
+  // Radius
+  ItemBorderRadius: theme.BorderRadiusBase,
+
+  // Space
+  SpaceMarginBlockEnd: theme.SpaceRemXs,
+  SpaceMarginInlineStart: theme.SpaceRemXs,
+  ItemSpaceMarginBlockStart: 0,
+  ItemSpaceMarginBlockEnd: theme.SpaceRemXs,
+  ItemSpaceMarginInlineStart: 0,
+  ItemSpaceMarginInlineEnd: theme.SpaceRemXs,
+  ItemSpacePaddingBlock: 0,
+  ItemSpacePaddingInline: theme.SpaceEmXs,
```

Comment:
one of the things I like about adding these name "parts" as a suffix (as opposed to a prefix here) is that they occupy the same part of the variable name that variants do - so all the tokens become more uniform and consistent across the API!

e.g. `SpacePaddingInlineItem`

Reply:
I don't mind making that change, your argument is valid. I prefer it as a prefix while developing but for the consumer it makes more sense as a suffix.

Y'all will need to agree on one way or the other 😆